dll/win32/shimgvw/shimgvw.c

```diff
@@ -949,72 +975,60 @@ ImageView_WndProc(HWND hwnd, UINT Message, WPARAM wParam, LPARAM lParam)
             return 0L;
         }
 
-        case WM_KEYDOWN:
-            switch (LOWORD(wParam))
-            {
-                case VK_LEFT:
-                    PostMessage(hwnd, WM_COMMAND, MAKEWPARAM(IDC_PREV, BN_CLICKED), (LPARAM)NULL);
-                    break;
-
-                case VK_RIGHT:
-                    PostMessage(hwnd, WM_COMMAND, MAKEWPARAM(IDC_NEXT, BN_CLICKED), (LPARAM)NULL);
-                    break;
-            }
-            break;
-
         case WM_COMMAND:
         {
-            switch (wParam)
+            switch (LOWORD(wParam))
             {
-                case IDC_PREV:
-                {
+                case IDC_PREV_PIC:
                     currentFile = currentFile->Prev;
                     pLoadImageFromNode(currentFile, hwnd);
-                }
-                break;
+                    break;
 
-                case IDC_NEXT:
-                {
+                case IDC_NEXT_PIC:
                     currentFile = currentFile->Next;
                     pLoadImageFromNode(currentFile, hwnd);
-                }
-                break;
+                    break;
+
+                case IDC_BEST_FIT:
+                    break;
```

Comment:
Add `TODO:` comment here.

Reply:
This change has broken the patch. It says that WARN() is undefined.

Reply:
`#include <debug.h>`

Reply:
It seems to me that there is already this debug.h, on top of the source code.
At the moment I commented these lines, to make it working again.

Reply:
Can you use DPRINT1 instead of WARN?